### PR TITLE
add community call from 2024-09-19

### PR DESCRIPTION
**Pull request by fkr**

### CODE REVIEW

File: community-call/20240919.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+- vPTG is upcoming: 2024-10-21 through -25 virtual
+- Update on Public-Cloud SIG: changed weeks
+- Lean Operator coffee next monday: talk standardization
+
+----
+
+## Team IaaS
+
+part 2/2
```
Comment (mbuechse) [inline community-call/20240919.md:80]:
```suggestion
part 2/3
```

File: community-call/20240919.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+  - how can S3 endpoints be discovered when Swift isn't available?
+    - be pragmatic: try Swift endpoints or libs3 environment vars
+  - how old are service APIs allowed to be?
+    - question calls for another standard
+
+----
+
+## Team IaaS
+
+part 3/2 (gotcha!)
```
Comment (mbuechse) [inline community-call/20240919.md:94]:
```suggestion
part 3/3
```

File: community-call/20240919.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+- Upstream contribution of subnet NTP server injection
+  - discuss in Public-Cloud SIG
+- DNS Standard: request for comments
+
+
+---
+
+# Team IAM & Security
+
+Report by ? <a href="https://github.com/garloff">@garloff</a>
```
Comment (mbuechse) [inline community-call/20240919.md:107]:
```suggestion
Report by <a href="https://github.com/garloff">@garloff</a>
```

**Comment by mbuechse** [comment]
@fkr  If you want to keep the binaries, then you should actually reference them instead of the URLs from the HedgeDoc instance.

File: community-call/20240919.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+## <font color="#0f5fe1" style="text-shadow: -1px 1px 0 #FFF, 1px 1px 0 #FFF, 1px -1px 0 #FFF, -1px -1px 0 #FFF;">Community Call 2024-09-19</font>
+
+![](https://input.scs.community/uploads/a46465b4-dd13-45ec-803c-b53912a9cae9.jpg)
+
+
+---
+
+![](https://input.scs.community/uploads/ae131de0-65bb-41b6-80b4-792a51f33961.png)
+
+https://en.wikipedia.org/wiki/International_Talk_Like_a_Pirate_Day
```
[the remark this answers is not in the document]
Comment (fkr) [reply community-call/20240919.md]: thanks. I removed the url and only referenced them locally.

File: community-call/20240919.md (hunk 1 above)
Comment (fkr) [reply community-call/20240919.md:80]: I want to add the slides in the way they were presented. Why adjust this?

Comment (fkr) [reply community-call/20240919.md:80]: well, actually. I don't care :)

File: community-call/20240919.md (hunk 3 above)
Comment (fkr) [reply community-call/20240919.md:107]: thanks - I left that as I usually store these away the way they were presented.

**Comment by fkr** [comment]
> @fkr If you want to keep the binaries, then you should actually reference them instead of the URLs from the HedgeDoc instance.

yes. thanks for spotting that! fixed. Sorry for wasting time with this :)

**Pull request merged**